ui/src/tool-panel/location-track/dialog/location-track-edit-dialog-name-section.tsx

```diff
@@ -248,10 +257,15 @@ export const LocationTrackEditDialogNameSection: React.FC<
                             onCommitField={onCommitField}
                             getVisibleErrorsByProp={getVisibleErrorsByProp}
                         />
-                        <FieldLayout
-                            label={`${t('location-track-dialog.full-name')}`}
-                            value={fullName}
-                        />
+                        {state.locationTrack.namingScheme &&
+                            !NAMING_SCHEMES_WITHOUT_FULL_NAME.includes(
+                                state.locationTrack.namingScheme,
+                            ) && (
```

Comment:
Hieman ärsyttävä ehto johtuen siitä että `LocationTrackSaveRequest`-tyyppi sallii sen, että `namingScheme` on `undefined` ja se pitää siksi tarkistaa erikseen. Tämän saisi varmaankin kierrettyä tyypittämällä tuon taulukon mahdollistamaan myös `undefined`:it sisällensä, mutta mielestäni on parempi että tyypit on mahdollisimman oikein kuin että täältä saa yhden ehdon pois